common/unsafe/src/main/java/org/apache/spark/sql/catalyst/util/CollationFactory.java

```diff
@@ -217,7 +217,7 @@ public Collation(
      * bit 31-24: Zeroes.
      * bit 23-22: Zeroes, reserved for version.
      * bit 21-18: Reserved for space trimming.
-     * 0000 = none, 0001 = left trim, 0010 = right trim, 0011 = trim.
+     * 0000 = none, 0001 = right trim. Bits 21-19 remain reserved and fixed to 0.
```

Comment:
nit: Could you separate this in two parts. bit 18 and then 0 and 1 and 21-19 reserved. For readability.

Reply:
Done.